kokoro/builds/build.sh

```diff
@@ -78,6 +78,11 @@ if [ -n "$1" ]; then
   }
   trap cleanup EXIT
 
+  # Temporary Ninja hack - will be removed when we move to Ninja in general
```

Comment:
Should you create a bug for that and add it here as a TODO?

Reply:
done.